src/fam.rs

```diff
@@ -938,4 +959,55 @@ mod tests {
         type Message2FamStructWrapper = FamStructWrapper<Message2>;
         assert!(serde_json::from_str::<Message2FamStructWrapper>(data_ser.as_str()).is_err());
     }
+
+    #[test]
+    fn test_clone_multiple_fields() {
```

Comment:
It would be nice to also test the case where we drop the original structure and make sure that the clone remains working. Just to make sure that the new structure doesn't point to something in the old one. Normally it shouldn't happen, but we're doing some unsafe things.